source/gui/settingsDialogs.py

```diff
@@ -1116,7 +1116,28 @@ def makeSettings(self, settingsSizer):
 		#
 		self.lineIndentationCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report l&ine indentation"))
 		self.lineIndentationCheckBox.SetValue(config.conf["documentFormatting"]["reportLineIndentation"])
+		self.lineIndentationCheckBox.Bind(wx.EVT_CHECKBOX,self.onReportIndent)

```

Comment:
nit: Maybe `self.onReportIndentChange` to be clearer? We don't do `onBlahClick` for buttons, but that's because an "action" seems natural without the clarification.